x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/assignment/TrainedModelAssignmentMetadata.java

```diff
@@ -129,7 +129,7 @@ public String getWriteableName() {
 
     @Override
     public Version getMinimalSupportedVersion() {
-        return Version.V_8_3_0;
+        return Version.V_8_0_0;
```

Comment:
This prevents the object from being serialized to older nodes. Since we don't update the object metadata custom name until all nodes are upgraded now, this is OK.